tools/pre-commit

```diff
@@ -9,11 +9,16 @@ JSHINT_EXECUTABLE="node_modules/.bin/jshint"
 JSHINT_XFAIL_LIST="build/jshint/xfail.list"
 JSHINT_ARGS="--reporter=build/jshint/xfail $JSHINT_ARGS"
 excluded_list=".jshintignore"
+L10N_XFAIL_LIST="build/l10n/xfail.list"
```

Comment:
At first I thought we'd start small, with an empty file, and grow it as needed, but I ended up adding the full list.